### DISCUSSION TITLE
Place account information into Order command

### DISCUSSION THREAD

**Discussion by trta911** [maxicus/ib-tws-api#9] [General]
Hi, we've got message from Interactive brokers that need from us for successfull order placement to include an account number to order. So how can I do that? It can be achieved by code bellow please? (it is not covered by the docs, I think):

```
var order = {
  action: 'BUY',
  totalQuantity : 1, 
  account: U123456F
};

 ib.placeOrder(1, ib.contract.cfd('AAPL', 'SMART', 'USD'), order);
```

**Comment by maxicus** [comment]
yes. with quotes ofc

**Comment by tredondo** [comment]
Yes, see [Account](https://interactivebrokers.github.io/tws-api/classIBApi_1_1Order.html#a8edb7e614aa530a58c647d8d273b1d8b) field of the `Order` type. It's a string.
